mrp_workcenter_account_move/tests/test_mrp_production.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
```

Comment:
The new standard accepts: `# coding: utf-8` or `# -*- coding: utf-8 -*-` to avoid red pylint in CI-Travis.